Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -5021,4 +5021,87 @@ final class BuildPlanTests: XCTestCase {
             XCTFail("expected a Swift target")
         }
     }
+
+    func testBasicSwiftPackageWithoutLocalRpath() throws {
```

Comment:
This test is a copy and paste of `testBasicSwiftPackage()` above, with the rpath disabled and a few checks removed.